services/dynamic-sidecar/openapi.json

```diff
@@ -557,6 +557,96 @@
           }
         }
       }
+    },
+    "/v1/containers/{id}/networks:attach": {
```

Comment:
QUESTION: how useful you find having the openapi.json always in the repo ( little use /useful/very useful?) . I would like to rethink how this can be created...

Reply:
Well, for this service, because it is enforced, I can say 100% of the times that the specs are up to date with the definitions if the unit tests don't fail. Other than that I do not use it. I prefer looking at the FastAPI signatures.
For the webserver instead it is much more helpful to use the openapi spec.